serverless/reference/kn-event-ref.adoc

```diff
@@ -0,0 +1,12 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="kn-event-ref"]
+= kn event
+:context: kn-event-ref
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+You can use the `kn event` set of commands to manage cloud events from the command line.
```

Comment:
Per https://github.com/openshift/openshift-docs/pull/40821/files#diff-c5f7fa7193e0ac8c82c3a1a2b194609175ee251efe1e4e94fc0ace57e6bf5a0bR21 I think this is tech preview? In which case it needs the tech preview module / snippet included here.